<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_qty_part</class>
 <widget class="QDialog" name="add_qty_part">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADD PART QUANTITY</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>491</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="invoice_num">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INVOICE NUMBER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_invoiceNum">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
    animation: blink 0.5s alternate infinite; /* Add a blinking animation */
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>431</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="qtyAdd">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>QTY TO ADD</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_QTY_add">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
    animation: blink 0.5s alternate infinite; /* Add a blinking animation */
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>481</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="manufacturer">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MANUFACTURER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_manufacturer">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
    animation: blink 0.5s alternate infinite; /* Add a blinking animation */
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>371</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="remark">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>REMARK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_Comments">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
    animation: blink 0.5s alternate infinite; /* Add a blinking animation */
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>391</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="partNum">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PART NO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_partNo">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
    animation: blink 0.5s alternate infinite; /* Add a blinking animation */
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>381</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PROJECT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_projj">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
    animation: blink 0.5s alternate infinite; /* Add a blinking animation */
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QPushButton" name="pushButton_add_new_proj">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{

	

	font: 18pt &quot;Times New Roman&quot;;  

		border-right: 1px solid #aaaaaa;

		border-bottom: 1px solid #aaaaaa;



}



QPushButton:enabled {

		background-color: green;

		color: white;

}



QPushButton:pressed {

		background-color: green;

		color: white;

}





QPushButton:hover:!pressed {

		background-color: ORANGE;

		color: red;

}



QPushButton:disabled {

		background-color: #aaaaaa;

		color: #ffffff;

}</string>
      </property>
      <property name="text">
       <string>Add New Project</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ref">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="asert.qrc">
        <normaloff>:/Images/refresh.png</normaloff>:/Images/refresh.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_addPart">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="asert.qrc">
        <normaloff>:/Images/submit.png</normaloff>:/Images/submit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="asert.qrc">
        <normaloff>:/Images/close.png</normaloff>:/Images/close.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_calender">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>48</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{

	

	font: 18pt &quot;Times New Roman&quot;;  

		border-right: 1px solid #aaaaaa;

		border-bottom: 1px solid #aaaaaa;



}



QPushButton:enabled {

		background-color: green;

		color: white;

}



QPushButton:pressed {

		background-color: green;

		color: white;

}





QPushButton:hover:!pressed {

		background-color: ORANGE;

		color: red;

}



QPushButton:disabled {

		background-color: #aaaaaa;

		color: #ffffff;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="asert.qrc">
     <normaloff>:/Images/date.png</normaloff>:/Images/date.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INVOICE DATE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>142</y>
     <width>231</width>
     <height>47</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}

QLineEdit:hover {
    background-color: #e6f3ff; /* Set the hover background color to light blue */
    animation: blink 0.5s alternate infinite; /* Add a blinking animation */
}

@keyframes blink {
    from {
        background-color: #e6f3ff;
    }
    to {
        background-color: #ffffff;
    }
}
</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="asert.qrc"/>
 </resources>
 <connections/>
</ui>
